release.yaml

```diff
@@ -0,0 +1,14 @@
+kuadrant-operator:
```

Comment:
I wonder if this file should be named something like `release.yaml.tmpl` or `release.yaml.in` in the `main` branch, and renamed/duplicated as `release.yaml` only in the release branches.

Reply:
Mainly communication, to call the file for what it is. This file should only exist in release branches. The only meaning it has in `main` is as a template.

Plus, we don't want people modifying and committing this file in `main`. Some distraction and it may end up with some of values set something other than `0.0.0`, which is ugly, to say the least.

Reply:
I would not say it should only exist on release branches, there are possible advantages to how the nightly pipelines could make use of the file. None of that work has being done as of yet. There is also a hope to get `make local-setup` to be able to consume that file, so it is not truly  about releases.